packages/application-shell/src/components/application-shell-provider/application-shell-provider.tsx

```diff
@@ -50,7 +49,6 @@ const ApplicationShellProvider = <AdditionalEnvironmentProperties extends {}>(
   const browserLocale = getBrowserLocale(window);
   return (
     <>
-      <GlobalStyles />
```

Comment:
This has been moved to the `ApplicationShell` component to have all the global styles in the same place.
[This is the comment](https://github.com/commercetools/merchant-center-application-kit/pull/2839#discussion_r985597519) where I explain the motivation a little more.

Reply:
As discussed with @emmenko, this change might break an edge case we have in the FE with the authentication app since it uses the `ApplicationShellProvider` directly and does not use the `ApplicationShell` component.
We agreed this change makes sense but I'll need to keep an eye on that use case to check whether there will be a problem.